### DISCUSSION TITLE
No OLED.... No power on?  Power is comming to PCP bord...

### DISCUSSION THREAD

**Discussion by Vek001** [sanni/cartreader#233] [Need help]
Hi
This is great project.... Big THX for sanni and everyone.

Now I can make cartreader well.
Today I made 3 cartreader, all good work.
I think point is the soldering of the GBA... 

However, in the process I have created many cartreaders with the same symptoms that don't work.

No OLED,  LED is dark and red, nothing changes when you insert the SD card and press the button.
When I switch between 5V and 3V, the brightness changes.

So, I change OLED ,re-soldering of Arduino pin , change register and check.....
Checked the voltage too
![IMG_3032](https://user-images.githubusercontent.com/88176907/130475014-e6e49df1-70d4-4b3f-ad36-5075fa3de3ab.jpg)
![IMG_3045](https://user-images.githubusercontent.com/88176907/130475035-3d0a2ee5-a277-499d-8af3-837acfe75d83.jpg)
![IMG_3051](https://user-images.githubusercontent.com/88176907/130475047-ffd60cb3-7b51-4d30-8ab1-28eb106506e2.jpg)

I have a lot of cartreaders with the same problem as this one.
I feel sorry for these kids and want to fix them.
Is there anything I can check or adjust?
![IMG_3050](https://user-images.githubusercontent.com/88176907/130475130-9945a0cd-4b2d-417d-90c3-783ccdd026cb.jpg)

**Comment by Dakkaron** [comment]
Did you enable the OLED in the configuration.h?

**Reply by Vek001** [reply]
Hi Dakkaron.
Thank you!

Configuration?
You mean the Arduino configuration?
I did not any change configuration.
Some of them have been made the same way and have worked well.

**Reply by Dakkaron** [reply]
Sorry, the file is called options.h

There you can switch between OLED and Serial output. If the statement `#define enable_OLED` is deleted or commented out (\\ before the #define), the OLED is disabled.

**Reply by Vek001** [reply]
Oh, Thank you again!! 
Ya! I enable_OLED.
I've replaced it with a PCB board that works fine, and it screens fine.
![スクリーンショット 2021-08-24 021049](https://user-images.githubusercontent.com/88176907/130488678-aa2ffad3-5f72-4de5-b5d8-61c8c2b179f5.png)

**Comment by sanni** [comment]
Do the working Cart Readers have the exact same OLED module as the ones that are not working?

**Reply by Vek001** [reply]
Hi sanni.
Thank you so much.

YES.
It is all same OLED.

Today I will make new one.
and I'd also like to add a non-working OLED to my new kit.

**Reply by Vek001** [reply]
Hello
I made three more and all three work well
and two of them I used the old(non-working)OLED.

**If there are no more items to check...**
...Maybe these are better off as parts.
sorry sanni and PCB board.
And I'll burn the PCB in a frying pan and try to rescue the slots and fail.
